main.tf

```
@@ -108,7 +108,7 @@ resource "github_branch_protection" "repositories" {
   for_each          = var.repositories
   repository_id     = github_repository.repositories[each.key].node_id
   pattern           = "*"
-  push_restrictions = []
+  push_restrictions = [github_team.teams["continuous-integration"].node_id]
```

```suggestion
  push_restrictions = [data.github_user.mage-os-ci.node_id]
```

For mirrors, we allowed the `mage-os-ci` user explicitly (see line 88). I think we should do the same here.

---

Fixed in a48d04414ea51b55b7dc32041a95b20bfb5f61ac